texar/torch/modules/encoders/bert_encoders.py

```diff
@@ -41,15 +41,16 @@ class BERTEncoder(EncoderBase, PretrainedBERTMixin):
     :class:`~texar.torch.modules.encoders.TransformerEncoder` and a dense
     pooler.
 
-    This module supports the architecture first proposed
-    in `(Devlin et al.)` BERT.
+    This module supports the architectures first proposed
+    in `(Devlin et al.)` BERT and `(Liu et al.)` RoBERTa.
```

Comment:
Since multiple places refer to these BERT variants, we can consider putting the structured note I mentioned above to one place, and point users to it. Is there sth like `BERTBase` class for us to put this "meta" info there?